app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -981,3 +1248,4 @@ class RecentlyPlayedFragmentTest {
     override fun getApplicationInjector(): ApplicationInjector = component
   }
 }
+
```

Comment:
Nit: remove extra newline (only one is needed at the end of the file).

Reply:
Done